projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -93,12 +103,37 @@ const Index = () => {
 			>
 				<CriticalCssMeta />
 
-				<UpgradeCTA
-					identifier="critical-css"
+				<ProductInterstitialMyJetpack
```

Comment:
From I see in https://github.com/Automattic/jetpack-roadmap/issues/2060, all CTAs will be updated to use this interstitial.

I think it makes sense to move it to its own component to keep the index clean. I think the only thing we need to pass to the new component is the identifier. In this case `critical-css`.

Reply:
React isn't my first language, so I'm not sure if this can be handled differently.

From what I understand, this component will be added multiple times on the page (based on the number of UpgradeCTA at the moment).

Doesn't it make sense to have the component only once on the page and call it via the UpgradeCTA or something?

Reply:
>Doesn't it make sense to have the component only once on the page and call it via the UpgradeCTA or something?

In this case, it's a big deal since the Modal is not being rendered multiple times. We went with that approach to have a plug and play component to display the interstitial, without having to change that much of the existing components (CTAs) that linked to the interstitial page.